app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,17 @@
+class BookingsController < ApplicationController
+def index
+
+end
+def create
+
+end
+def update
+
+end
+def edit
```

Comment:
We don't need this, a button will trigger update (so the edit form is unnecessary)

Reply:
we need update action